code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -52,6 +52,13 @@
 			..()
 		return
 
+/obj/item/weapon/storage/lockbox/can_be_inserted(obj/item/W as obj, stop_messages = 0)
+	if(!(locked))
+		return ..()
+	if(!(stop_messages))
+		usr << "<span class='notice'>[src] is locked!</span>"

```

Comment:
No. This is an override for lockboxes, so only lockboxes call it, other containers still use the parent proc